CMakeLists.txt

```diff
@@ -94,15 +94,18 @@ endif()
 # Add web application
 # -----------------------------------------------------------------------------
 option(TOMVIZ_DOWNLOAD_WEB  "Enable downloading web application." OFF)
-set(tomviz_html_path "${CMAKE_BINARY_DIR}/web/tomviz.html")
+set(tomviz_html_source_path "${CMAKE_SOURCE_DIR}/web/tomviz.html")
```

Comment:
Does that matter isn't it the install location that is important? I am happy to change it but would like to understand why?